Adding new components and refactoring

iansolano opened the pull request:
### Status
**READY**

### Description
**FEATURE/REFACTOR**

Adding components and refactoring styles to support tutor payments work.

### Related PRs
https://github.com/hiclark/tutor-payments/pull/238

### Quality Checklist:
These are mandatory and must be completed before merging the PR

- [x] New, changed or refactored functionality has tests.
- [ ] New, changed or refactored functionality has documentation.
- [x] New, changed or refactored functionality has been QA'd.
- [x] New, changed or refactored functionality has been added to the CHANGELOG.

Jexeones24 commented on src/button/styles.js:
```diff
@@ -0,0 +1,151 @@
+
+const secondary = css`
+  max-width: 100%;
+  width: 100%;
```
Why do we need both if they are the same?

iansolano replied:
shouldn't

Jexeones24 commented on src/button/styles.js:
```diff
@@ -0,0 +1,151 @@
+
+  &:hover {
+    border: ${BW_1} solid transparent;
+    color: ${disabled ? WHITE : WHITE};
```
Is this right?

iansolano commented:
Button is a component that really needs and will get a totally refactor. As I am addressing your feedback @Jexeones24 I am coming across lots of issues but going to leave alone for now since we can tackle later when we refactor to accommodate new styles etc (very soon).

iansolano commented:
@Jexeones24 good to go?
